airbyte-protocol/protocol-models/src/main/resources/airbyte_protocol/well_known_types.yaml

```diff
@@ -73,3 +73,6 @@ definitions:
   Boolean:
     type: boolean
     description: Note the direct usage of a primitive boolean rather than string. Unlike Numbers and Integers, we don't expect unusual values  here.
+  ArbitraryJson:
+    type: object
```

Comment:
we don't need the `type` at all. I.e. the entire declaration would be:
```yaml
ArbitraryJson:
  description: .......
```
This is equivalent to a JSON schema of `{"description": "....."}`, which would accept any valid JSON value, e.g. `"any_string"`, `42`, `["array", {"foo": "bar"}]`, etc

(this is why we have to make the protocol update at all - otherwise we could just treat `{type: object}` schemas as JSONB)